static/compiler.js

```diff
@@ -361,6 +361,12 @@ Compiler.prototype.getEffectiveFilters = function () {
     if (filters.execute && !this.compiler.supportsExecute) {
         delete filters.execute;
     }
+    var disabledFilters = this.compiler.disabledFilters;
+    disabledFilters.forEach(function (filter) {
```

Comment:
Just in case, use `_.each()` instead of `.forEach` to ensure we support older browsers